src/test/java/com/axoniq/someproject/something/SomeAggregate.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2010-2023. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.axoniq.someproject.something;
+
+import com.axoniq.someproject.api.ChangeStatusCommand;
+import com.axoniq.someproject.api.SomeCommand;
+import com.axoniq.someproject.api.SomeEvent;
+import com.axoniq.someproject.api.StatusChangedEvent;
+import org.axonframework.commandhandling.CommandHandler;
+import org.axonframework.eventsourcing.EventSourcingHandler;
+import org.axonframework.modelling.command.AggregateIdentifier;
+import org.axonframework.spring.stereotype.Aggregate;
+
+import java.util.Objects;
+
+import static org.axonframework.modelling.command.AggregateLifecycle.apply;
+
+@Aggregate
```

Comment:
Although normally redundant when using `@Aggregate`, can we add `@AggregateRoot` here too? For testing purposes, of course.

Reply:
What is it we want to test about it? Just that even if it also has that annotation, it still works?

Reply:
I understand now from another conversation, will add it in the next commit.